.pre-commit-config.yaml

```diff
@@ -3,7 +3,6 @@ repos:
     rev: stable
     hooks:
     - id: black
-      language_version: python3.7
```

Comment:
Is this change on purpose? Just asking as it seems unrelated to the rest of this PR.

Reply:
the pre-commit hook was failing because I'm using python 3.8